src/components/Firestore/CollectionFilter/CollectionFilter.test.tsx

```diff
@@ -94,6 +94,13 @@ it('unsets sorting when switching to an operator that does not support it', asyn
 });
 
 it('supports multi-value operators', async () => {
+    // Stub the browser property performance.now()
+    Object.defineProperty(window, 'performance', {
```

Comment:
Not sure exactly how these tests run, so this might be misguided, but double check that this does not leak into other tests

Reply:
Ack - did some testing and it doesn't seem to affect other tests - I think this property is mostly unused as this is the only test that requires it